lib/spack/spack/cmd/external.py

```diff
@@ -26,12 +28,18 @@ def setup_parser(subparser):
     sp = subparser.add_subparsers(
         metavar='SUBCOMMAND', dest='external_command')
 
-    find_parser = sp.add_parser('find', help=external_find.__doc__)
+    find_parser = sp.add_parser(
+        'find', help='add external packages to packages.yaml'
+    )
     find_parser.add_argument(
         '--not-buildable', action='store_true', default=False,
         help="packages with detected externals won't be built with Spack")
     find_parser.add_argument('packages', nargs=argparse.REMAINDER)
 
+    sp.add_parser(
+        'list', help='list detectable packages, by repository and name'
+    )
```

Comment:
We should add a `spack external remove` command to remove entries based on spec. Doesn't need to be done in this PR though.

Reply:
I would definitely save this for a following PR :sweat_smile: